### DISCUSSION TITLE
Error when creating environment teams with Power Platform CLI - pac

### DISCUSSION THREAD

**Discussion by serjdaza** [microsoft/powerplatform-build-tools#735] [Q&A] [answered]
Hello, guys. I'm trying to create an environment Team with security role assigned with the command `pac admin assign-group` but I am confused with the parameters `--environment` and `--role`. How can I get these values? I have tried with environment id and passing "Environment Maker", "Basic User", etc, but always I get the following error: "Environment Maker security role not found in xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx environment". I tried passing the org ID in the `--environment` parameter but after some time stucked it throws the error "Sorry, the app encountered a non recoverable error and will need to terminate. The exception details have been captured and will be forwarde6-21bfc3ebae29, Exception Type: bolt.authentication.http.AuthenticatedClientException
The diagnostic logs can be found at...". Anybody could help please?
![image](https://github.com/microsoft/powerplatform-build-tools/assets/86273422/bd02217d-8ea6-4b64-9950-577e680606e4)
![image](https://github.com/microsoft/powerplatform-build-tools/assets/86273422/17d2a63e-68ff-4560-9301-34161357f401)

**Comment by elissa-williams** [comment]
Did you ever find a solution?

**Reply by serjdaza** [reply, marked as answer]
Yes, I did! The problem was not the `--environment` nor `--role` parameters but the `--business-unit` instead. `--environment` is the same that you can see in the environment details in admin center (see picture below), and `--role` is the name of the security role (in my example is "Environment Maker").
![image](https://github.com/user-attachments/assets/34b8bf0c-2b9a-4644-8a31-c22b7c32fc71)
The `--business-unit` parameter, in the other hand, was a bit tricky to find but is basically the id of the business unit. You can find it in the environment table called "Business Unit" and the column is "businessunitid". See picture below :)
![image](https://github.com/user-attachments/assets/68f49ded-27d6-4c4b-a42a-4b83baac28c0)

**Reply by elissa-williams** [reply]
Thank you so much! Hopefully that will solve my issue too
